db/migrate/20170706152142_add_category_option_id_to_categories.rb

```diff
@@ -0,0 +1,8 @@
+class AddCategoryOptionIdToCategories < ActiveRecord::Migration
+  def change
```

Comment:
You should change this from `def change` to `def up` and `def down` like in the examples section of http://api.rubyonrails.org/classes/ActiveRecord/Migration.html